### ISSUE TITLE
Firmware AX5400 CB401 ?

### ISSUE THREAD

**Issue by Yoshi1981** [openwrt-xiaomi/xmir-patcher#17]
Firmware AX5400 CB401 ?

No Found

**Comment by remittor** [comment]
CB0401
https://cdn.awsde0-fusion.fds.api.mi-img.com/xiaoqiang/rom/cb0401/miwifi_cb0401_R01A11-FOTA_firmware_b173d_3.0.100_INT.bin

**Comment by Yoshi1981** [comment]
Thanks

**Comment by Yoshi1981** [comment]
Can Flash original Firmware this bin or only With patcher ? Is this for all CB401 (Magenta T-Mobile edition) thanks

**Comment by lanke521** [comment]
> 可以刷入原始固件吗？还是只能使用补丁程序？这适用于所有 CB401（Magenta T-Mobile 版）谢谢
The firmware you flashed into my model is cb401v2. Can I update the firmware directly in the system?

**Comment by lanke521** [comment]
> 只能使用补丁程序？

The firmware you flashed into my model is cb401v2. Can I update the firmware directly in the system?

> https://cdn.awsde0-fusion.fds.api.mi-img.com/xiaoqiang/rom/cb0401/miwifi_cb0401_R01A11-FOTA_firmware_b173d_3.0.100_INT.bin

The firmware you flashed into my model is cb401v2. Can I update the firmware directly in the system?

**Comment by lanke521** [comment]
> https://cdn.awsde0-fusion.fds.api.mi-img.com/xiaoqiang/rom/cb0401/miwifi_cb0401_R01A11-FOTA_firmware_b173d_3.0.100_INT.bin


Hello, my model is cb401v2. Can I update the firmware directly in the system?

**Comment by lanke521** [comment]
> 可以刷入原始固件吗？还是只能使用补丁程序？这适用于所有 CB401（Magenta T-Mobile 版）谢谢

我的型号跟你一样你的刷进去了吗，可以直接刷吗有中文界面吗

**Comment by remittor** [comment]
CB0401
https://cdn.awsde0-fusion.fds.api.mi-img.com/xiaoqiang/rom/cb0401/miwifi_cb0401_R01A13-FOTA_firmware_dce0f_3.0.116_INT.bin

**Comment by marcoamx-png** [comment]
My router only have the power led on in orange, after 40s all lights blink white and the orange go back.
I have limited network by cable, but not access to landing page.

Can any one explain me, please, how can i try to install the firmware?

Or its garbage?

**Comment by tilator** [comment]
If the FW is updated to 3.0.116, is there any vulnerabilities or anything else valuable missed? Or is it possible to downgrade FW when ever needed?

And is there any documentation about how to SSH this device?

**Comment by marcoamx-png** [comment]
> My router only have the power led on in orange, after 40s all lights blink white and the orange go back. I have limited network by cable, but not access to landing page.
> 
> Can any one explain me, please, how can i try to install the firmware?
> 
> Or its garbage?

Any one? Pls

**Comment by tilator** [comment]
> Any one? Pls

It's in a bootloop. This is chinese garbage.

Where did you get it from?

**Comment by antonio123beluchi-cyber** [comment]
> > На моем роутере горит только оранжевый индикатор питания, через 40 секунд все индикаторы начинают мигать белым, а затем снова загорается оранжевый. У меня ограниченное кабельное подключение, но нет доступа к главной странице.
> > Может кто-нибудь объяснить, пожалуйста, как мне установить прошивку?
> > Или это мусор?
> 
> Кто-нибудь? Пожалуйста.

hi everyone, maybe you managed to fix this and somehow flash your router? I have the CB0401V2 model as well, with the exact same symptoms.

**Comment by Kreal-exe** [comment]
Yes — see https://github.com/Kreal-exe/cb0401-tune-control, it documents (and automates) exactly this: the default root/Telnet password is derived from the serial number your router exposes at the unauthenticated `api/xqsystem/init_info` endpoint (`md5(serial + <fixed GUID salt>)`, first 8 hex chars), and Telnet is always on by default on stock firmware. The toolkit opens SSH from that automatically, strips the telemetry/junk cron jobs the stock firmware phones home with, then gives you a local GUI for band tuning, firmware downgrade, etc. Downgrading is possible too — there's a one-click version-spoof that lifts the updater's downgrade block.
